e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -221,7 +221,7 @@ $verificationCertPath = "$PSScriptRoot/verification.cer";
 
 $iotHubX509DeviceCertCommonName = "iothubx509device1";
```

Comment:
This common name is used for the device Id for self-signed x509 enrollment. We'll want to append the "save" prefix here as well.

Reply:
This is just the cert name. 2 lines down is the name we use for the device.

Does it matter?